### DISCUSSION TITLE
Problem with Chinese VS1053 Board

### DISCUSSION THREAD

**Discussion by pschatzmann** [baldram/ESP_VS1053_Library#101] [Q&A] [answered]
Sorry to bother you, but I would not know who else to ask.
I bought the [following cheap VS1053 module](https://www.pschatzmann.ch/wp-content/uploads/2022/08/vs1053.jpg) from China but I only have problems: The SPI connection seems to work properly but if I send it some mp3 file, I only get some strange gargling noises. 

![vs1053](https://www.pschatzmann.ch/wp-content/uploads/2022/08/vs1053.jpg)

Would anybody know what this could be ?

I did not connect the CS pin on the board - but connected the CS from the microcontroller to the XCS on the board: assuming that the CS is meant to be for the SD card.

Another VS1003 based board is working properly with the same software...

ps. I am potentially looking for some cheap alternative VS1053 boards which support 3.3V just to test some software. 
Are there any suggestions?

**Comment by Dr-Dawg** [comment]
Hm, I always thought of CS == XS, did you try to use CS instead?
Which SPI pins do you use, standard VSPI = 5, 18, 19, 24, i.e. XS = 5? Did you try to change that, i.e. XS = 32?
Did you try the mp3 or the webradio demo and what does player.getChipVersion() return?

**Comment by pschatzmann** [comment, marked as answer]
The board is reporting as version 4, so it is a VS1053!

You are right on the VS1003 the CS pin is labeled XS. As you can see in the link this must be XCS on my board.
Connecting to CS breaks the SPI communication. 

I am using the Arduino SPI, so VSPI pins are 5, 18, 19, __23__!,

I re-mapped DCS to gpio 2 and thinks look better now: I get music but a lot of brownouts, but sometimes the music is playing.
That's strange because GPIO 5 was perfectly working as DCS for other modules!

So I might have an issue with the power supply...
